Makefile

```diff
@@ -15,6 +15,8 @@ clean-local-properties:
 	rm -f local.properties
 
 sdk: clean-local-properties
+	emulator -verbose -avd device1 -no-skin -no-audio -no-window&
+	sleep 30
```

Comment:
1. I don't think this is where you want this. This is `make sdk` which should only make the sdk. You do not need an emulator for that.

2. Sleep is a red flag. What are we waiting for? 30 is a magic number. Why 30? If we _have_ to use this we need to explain why. I would like to explore other options first.